src/main/java/io/github/nucleuspowered/nucleus/modules/core/commands/MigrateCommand.java

```diff
@@ -0,0 +1,130 @@
+import io.github.hsyyid.essentialcmds.api.util.config.Configurable;
+import io.github.hsyyid.essentialcmds.managers.config.HomeConfig;
+import io.github.hsyyid.essentialcmds.managers.config.JailConfig;
+import io.github.hsyyid.essentialcmds.utils.Utils;
+import io.github.nucleuspowered.nucleus.Util;
+import io.github.nucleuspowered.nucleus.api.data.MuteData;
+import io.github.nucleuspowered.nucleus.api.data.NucleusUser;
+import io.github.nucleuspowered.nucleus.api.service.NucleusWarpService;
+import io.github.nucleuspowered.nucleus.config.GeneralDataStore;
+import io.github.nucleuspowered.nucleus.config.loaders.UserConfigLoader;
+import io.github.nucleuspowered.nucleus.internal.annotations.NoCooldown;
+import io.github.nucleuspowered.nucleus.internal.annotations.NoCost;
+import io.github.nucleuspowered.nucleus.internal.annotations.NoWarmup;
+import io.github.nucleuspowered.nucleus.internal.annotations.Permissions;
+import io.github.nucleuspowered.nucleus.internal.annotations.RegisterCommand;
+import io.github.nucleuspowered.nucleus.internal.annotations.RunAsync;
+import io.github.nucleuspowered.nucleus.internal.command.CommandBase;
+import io.github.nucleuspowered.nucleus.internal.interfaces.InternalNucleusUser;
+import io.github.nucleuspowered.nucleus.modules.jail.handlers.JailHandler;
+import io.github.nucleuspowered.nucleus.modules.mute.handler.MuteHandler;
+import ninja.leaping.configurate.objectmapping.ObjectMappingException;
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.entity.Transform;
+import org.spongepowered.api.item.ItemType;
+import org.spongepowered.api.item.ItemTypes;
+import org.spongepowered.api.world.Location;
+import org.spongepowered.api.world.World;
+
+import java.io.IOException;
+import java.util.UUID;
+
+/**
+ * Saves the data files.
+ *
+ * Permission: nucleus.nucleus.migrate
+ */
+@RunAsync
+@NoCooldown
+@NoCost
+@NoWarmup
+@Permissions(root = "nucleus")
+@RegisterCommand(value = "migrate", subcommandOf = NucleusCommand.class)
+public class MigrateCommand extends CommandBase<CommandSource> {
+
+    @Inject private UserConfigLoader userConfigLoader;
+    @Inject private JailHandler jailHandler;
+    @Inject private MuteHandler muteHandler;
+
+    @Override
+    public CommandResult executeCommand(CommandSource src, CommandContext args) throws Exception {
+        if (Sponge.getPluginManager().getPlugin("io.github.hsyyid.essentialcmds").isPresent()) {
+            // Warps
+            for (Object warp : Utils.getWarps()) {
+                String warpName = String.valueOf(warp);
+                Transform<World> warpLocation = Utils.getWarp(warpName);
+
+                if (warpLocation != null) {
+                    NucleusWarpService qs = Sponge.getServiceManager().provideUnchecked(NucleusWarpService.class);
```

Comment:
If the warp module is disabled, there will be no `NucleusWarpService`, so this will fail.